demo/app.component.ts

```diff
@@ -85,60 +93,66 @@ export class AppComponent implements OnInit {
     children: [
       {
         value: 'Serif  -  All my children and I are STATIC ¯\\_(ツ)_/¯',
+        id: 1,
         settings: {
           'static': true
         },
         children: [
-          { value: '<a href="#" id="antiqua" class="test">Antiqua</a> with HTML tags.'},
```

Comment:
We should have a chunk of html in some node's value as an example, cause I've merged a pull request that allows to render node's value as html :)